lib/active_fedora/file.rb

```diff
@@ -196,7 +216,7 @@ def uploaded_file?(payload)
       end
 
       def local_or_remote_content(ensure_fetch = true)
```

Comment:
My understanding is that remote_content here means the version of the content that lives in fedora as opposed to the local yet to be persisted version of the content.  External content is different from those two and thus a new name.  It might be appropriate to call this method `:local_or_remote_or_external_content` now but that seems wordy and confusing.